packages/terra-time-input/src/terra-dev-site/doc/example/TimeInputSeconds.jsx

```diff
@@ -18,7 +22,7 @@ class timeInput extends React.Component {
       <div>
         <p>
           Time Provided:
-          <span style={{ display: 'inline-block' }}>{this.state.time}</span>
+          <span className={cx('time-wrapper')}>{this.state.time}</span>
```

Comment:
The changes made in here, TimeInputSecondsTwelveHour.jsx, and TimeInputSecondsWithDefault.jsx are being taken care of in #810. We can either get rid of these changes here or close #810 to keep these changes.